### ISSUE TITLE
nitrox with latest version of game

### ISSUE THREAD

**Issue by Trigger117** [WhiteRoseOfTheSea/Nitrox-Cracked#3]
### Describe the issue

How can i use the latest version of the game with your nitrox?

**Comment by WhiteRoseOfTheSea** [comment]
> ### Describe the issue
> 
> How can i use the latest version of the game with your nitrox? 

Hello, unfortunately you can't use the latest version of the game, the main reason is because Subnautica with the update 2.0 changed the actual code on the game so it broke nitrox that now works only with a certain version.

**Comment by Trigger117** [comment]
so even if I used the regular version of the game it wouldn't work the same with regular nitrox. We will therefore have to wait for nitrox to be updated to be able to "modify", right?

**Comment by WhiteRoseOfTheSea** [comment]
> Thank you for answer!  So even if I used the regular version of the game it wouldn't work the same with regular nitrox. We will therefore have to wait for nitrox to be updated to be able to "modify", right?

Exactly, we need to wait until nitrox will update and make compatibility with the latest version of the game, I don't know if the developers of nitrox will ever do it, but if they do I'll crack it and update this GitHub.

**Comment by Trigger117** [comment]
last thing, I saw that you modified nitrox for a fairly old version of the game, is it possible to modify it for a newer version?  I won't bore you with questions anymore, I promise ;)

**Comment by WhiteRoseOfTheSea** [comment]
> last thing, I saw that you modified nitrox for a fairly old version of the game, is it possible to modify it for a newer version?  I won't bore you with questions anymore, I promise ;)

I tried every single cracked version of Subnautica and this is the more updated one that works, I've done that research for 2 days, but you can try on your own and test various versions if you want.

**Comment by Trigger117** [comment]
really strange, strange how they manage to run the newer versions on the official nitrox... maybe I'll do some research too

**Comment by WhiteRoseOfTheSea** [comment]
> really strange, strange how they manage to run the newer versions on the official nitrox... maybe I'll do some research too

That's because they use the Legacy version of steam on the official nitrox, but when we are in the cracked Aerea we talk about the version of the game, the Legacy one is December 2021, version 68598, the one that is on this GitHub

**Comment by dartasen** [comment]
> > Thank you for answer!  So even if I used the regular version of the game it wouldn't work the same with regular nitrox. We will therefore have to wait for nitrox to be updated to be able to "modify", right?
> 
> Exactly, we need to wait until nitrox will update and make compatibility with the latest version of the game, I don't know if the developers of nitrox will ever do it, but if they do I'll crack it and update this GitHub.

@WhiteRoseOfTheSea FYI we sent a DMCA takedown for this repository and for you as well.
